test/fixtures/samples/css/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't think we need a sample for now, as it's very simple. You may remove it.

Reply:
removed the css-modules example folder